firmware/src/transceiver.c

```diff
@@ -95,6 +108,7 @@ typedef enum {
   OP_RDM_BROADCAST = T_OP_RDM_BROADCAST,
   OP_RDM_WITH_RESPONSE = T_OP_RDM_WITH_RESPONSE,
   OP_RX = T_OP_RX,
+  OP_SELF_TEST = T_OP_SELF_TEST,
   OP_RDM_DUB_RESPONSE,  //!< No break
   OP_RDM_RESEPONSE  //!< With a break
 } InternalOperation;
```

Comment:
Can you Doxygen this enum, or remove the blank @brief so we know it's not been documented.